karma.conf.js

```diff
@@ -1,9 +1,5 @@
 process.env.MOZ_HEADLESS = 1;
 
-if (!process.env.GITHUB_ACTION) {
-    process.env.CHROME_BIN = require('puppeteer').executablePath();
-}
-
```

Comment:
Tiny detail but perhaps we can mark the browser tests `CI only` in [TESTING.md][1] as well. Think this change might make running those locally fail. 

[1]: https://github.com/ethereum/web3.js/blob/1.x/TESTING.md#required-tests

Reply:
@cgewecke thx, I think I have it working locally now with https://github.com/ethereum/web3.js/pull/3609/commits/c5430edbfc3d79a932b43422d7a589a7398f7c50